### ISSUE TITLE
Lab shuffle: TestHand doesn't test tally

### ISSUE THREAD

**Issue by jos-rosenqvist-3812** [lunduniversity/introprog#301]
The third task of the shuffle lab says to first implement `tally` in `Hand` and to then test `Hand` using `TestHand`. However, `TestHand` doesn't call `tally` as far as I can see, neither directly nor indirectly. Is this an oversight? It doesn't make much sense to ask the student to only test code that already is finished, and the task doesn't say to modify `TestHand` so that it calls `tally`.

**Comment by bjornregnell** [comment]
Good catch - a remains from an older version I guess. This should be changed somehow but I'll leave it for now in favour of getting compedium2 to the next level.

**Comment by bjornregnell** [comment]
@MechaMagpie Here is the issue we talked about at the meeting previously this week. If you do something aboyt this, you can write comments here and refer to this issue in commits if you submit PR:s on this. Let me know if we can close it after fixes.
